packaging/standalone/standalone-enterprise/src/main/distribution/text/enterprise/UPGRADE.txt

```diff
@@ -0,0 +1 @@
+For upgrade instructions, please see http://neo4j.com/docs/operations-manual/current/#upgrading.
```

Comment:
@MishaDemianenko Yes, that would be idea. But there is no facility in the docs system at the moment for calculating the correct URL (requires a mapping to remove the point release from the version number). Jonatan is aware of this problem and plans to sort it out at some point, but this is how manual URLs are being constructed across the codebase at the moment.